napalm/junos/junos.py

```diff
@@ -235,14 +236,70 @@ def compare_config(self):
             return diff.strip()
 
     def commit_config(self, confirmed=None, message=None):
-        """Commit configuration."""
+        """
+        Commit configuration.
+
+        if confirmed is passed, random string is generated for the commit log
```

Comment:
How so? Is this something done by pyez?

Reply:
I think this refers to my comment above regarding `pending_commit_string = str(uuid.uuid4()).split('-')[0]` which I still consider we shouldn't do that. But this again depends on what we agree whether the commit revert should be limited to napalm or the broad system.

Reply:
We shouldn't create a random commit message if the user doesn't specify one. Basically, `compare_config()` should behave exactly the same way before and after this PR.

Reply:
@dbarrosop sure, I removed that part of the code, will also update the docstring. I am not sure what you mean with `compare_config()` however. It always behaved the same way regardless.

Reply:
Yeah, sorry, I meant `commit_config`.